addon.py

```diff
@@ -111,7 +112,7 @@ def download_file(id):
 
 @plugin.route('/live', name='play_live')
 def play_live():
-    fetch_url = live_json.format(lang=language[0].upper())
+    fetch_url = live_json.format(lang=language_live[0].upper())
```

Comment:
What is the difference between :

```python
'f'[0].upper()
# and
'fr'[0].upper()
```

Are you sure redeclaring a variable is useful ?